data/web/inc/vars.inc.php

```diff
@@ -42,11 +42,13 @@
 $MAILCOW_APPS = array(
   array(
     'name' => 'SOGo',
-    'link' => '/SOGo/'
+    'link' => '/SOGo/',
+    'description' => 'SOGo is a collaborative software (groupware) server.'
```

Comment:
The first one is good. Change the second one to "Roundcube is a web-based email client" (i.e. drop the SOGo reference and don't claim it's lightweight -- it's a pretty massive piece of PHP and JS after all).